packages/compass-aggregations/src/components/aggregation-side-panel/stage-wizard-use-cases/field-combobox.tsx

```diff
@@ -81,7 +81,6 @@ export const FieldCombobox = ({
       overflow="scroll-x"
       {...props}
       options={fields}
-      value={value}
```

Comment:
Looks like setting this value does not change the behavior of the combobox. Let me know if I miss something here.